app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str | Exception:
```

Comment:
Do not use raised `Exception` in the return type annotation, it is not supposed to be present according [to the documentation](https://peps.python.org/pep-0484/#exceptions). Remove `Exception` in the return type annotation.